test/boost/mutation_writer_test.cc

```diff
@@ -563,7 +563,12 @@ SEASTAR_THREAD_TEST_CASE(test_token_group_based_splitting_mutation_writer) {
 
     size_t partition_count = many_partitions();
 
-    auto muts = tests::generate_random_mutations(random_schema, partition_count).get();
+    auto muts = tests::generate_random_mutations(
+            random_schema,
+            tests::default_timestamp_generator(),
+            tests::no_expiry_expiry_generator(),
+            std::uniform_int_distribution<size_t>(partition_count, partition_count),
+            std::uniform_int_distribution<size_t>(1, 1)).get(); // only 1 row
```

Comment:
if we pick a random number $i$ with following distribution
$$P(i|a,b)=\frac{1}{b-a+1}$$

where $a = b = 1$, so $i \equiv 1$. i think there is no need to put `std::uniform_int_distribution<size_t>(1, 1)` here. are we trying to use it to explain the idea that "we are pick a random number"?

Reply:
actually I am just conforming to existing interfaces :-) 

there's an interface that accepts partition count (but not row count) as size_t and translates it into std::uniform_int_distribution<size_t>(partition_count, partition_count), so I just did the same.

Reply:
ahh, that explains. thank you! i should have read the function signature first. and now i understand the comment better.